test/classes/initializers/generics/phase1/accessThisBad.good

```diff
@@ -1,2 +1,2 @@
 accessThisBad.chpl:4: In initializer:
-accessThisBad.chpl:5: error: cannot pass "this" to a function in phase 1 of initialization
+accessThisBad.chpl:5: error: cannot pass "this" to a function before calling super.init() or this.init()
```

Comment:
I really like that this message (and others in this PR) go in the direction of getting away from the phrase "phase 1" in the error messages.